libunwind/src/UnwindRegistersRestore.S

```diff
@@ -753,14 +772,13 @@ DEFINE_LIBUNWIND_FUNCTION(__libunwind_Registers_arm64_jumpto)
   // context struct, because it is allocated on the stack, and an exception
   // could clobber the de-allocated portion of the stack after csp has been
   // restored.
-  ldr    c16,     [c0, #0x1f0]
-  ldp    c0, c1,  [c0, #0x000]  // restore c0,c1
-  mov    csp,c16                // restore csp
-#ifdef __ARM_MORELLO_PURECAP_BENCHMARK_ABI
-  and    x30, x30, #~1
-  ret    x30                    // jump to pc
+  ldr    c2,      [c0, #0x1f0]
+  add    c3, c0,  #0x210
+  ldp    c0, c1,  [c0, #0x000]
+#ifdef _LIBUNWIND_SANDBOX_HARDENED
+  b      _rtld_unw_setcontext
 #else
-  ret                           // jump to pcc
+  b      _rtld_unw_setcontext_unsealed
```

Comment:
If _LIBUNWIND_SANDBOX_OTYPES is off surely we can just act like before rather than needing to make a function call?

Reply:
I think so. I was just avoiding adding code duplication since the thing we'd basically do is the same thing that's in the function call. Alternatively I could simply `#ifdef _LIBUNWIND_SANDBOX_OTYPES` the new code and then `#else` whatever we had there before to keep the functionality 1:1 when the define is not specified, but it would add some duplication.